x-pack/plugins/global_search_bar/public/components/search_bar.tsx

```diff
@@ -222,15 +217,15 @@ export const SearchBar: FC<SearchBarProps> = ({
         if (type === 'application') {
           const key = selected.key ?? 'unknown';
           const application = `${key.toLowerCase().replaceAll(' ', '_')}`;
-          trackUiMetric(
-            METRIC_TYPE.CLICK,
-            getClickMetric(CLICK_METRIC.USER_NAVIGATED_TO_APPLICATION, application)
-          );
+          trackUiMetric(METRIC_TYPE.CLICK, CLICK_METRIC.USER_NAVIGATED_TO_APPLICATION, {
```

Comment:
Changed the trackUiMetric function to combine context into the `eventName` internally, rather than having a `getClickMetric` function combine context.